specs/binding-filesystem.spec/src/main/resources/META-INF/zilla/filesystem.idl

```diff
@@ -31,13 +34,16 @@ scope filesystem
             FILE_NOT_FOUND,
             FILE_ALREADY_EXISTS,
             FILE_MODIFIED,
-            FILE_TAG_MISSING
+            FILE_TAG_MISSING,
+            DIRECTORY_ALREADY_EXISTS,
```

Comment:
I think we can remove `_ALREADY` from `DIRECTORY_ALREADY_EXISTS` and the equivalent `FILE_ALREADY_EXISTS`, giving `DIRECTORY_EXISTS` and `FILE_EXISTS` instead, agree?